code/socialDistribution/models/post.py

```diff
@@ -261,7 +266,10 @@ def get_id(self):
         return f"{API_BASE}/author/{self.author.id}/posts/{self.id}"
     
     def get_local_shareable_link(self):
-        return f"{CLIENT_BASE}/posts/local/{self.id}"
+        try:
+            return f"{CLIENT_BASE}/public-share/{self.id}" # publicly accessible (doesn't have to login)
+        except:
+            return f"{CLIENT_BASE}/posts/local/{self.id}" # only accessible to logged in users
```

Comment:
I'm not sure what you are trying to do here, could you please explain? The try clause will never throw any errors that need to be caught.

Reply:
it's the link that is only accessible to logged in users, but i got rid of it just to make things easier